src/store/pendingTxsSlice.ts

```diff
@@ -11,6 +12,7 @@ export enum PendingStatus {
 
 export type PendingTx = {
   chainId: string
+  safeAddress: string
```

Comment:
Safes that have transactions pending before we update could run into unexpected errors as the existing pending txs (before we release) will not have the `safeAddress` set.

Reply:
Yeah, I've tested this locally. It will basically not display these txs in the pending queue on top of the normal queue and it won't block immediate execution. IMHO that's fine, they disappear after a while.